gradle.properties

```diff
@@ -1 +1,9 @@
-kotlin.code.style=official
\ No newline at end of file
+kotlin.code.style=official
+
+#properties used for releasing to maven central
+signing.keyId=use gpg signing key
+signing.password=use correct signing pass
+signing.secretKeyRingFile=path to correct ../.gnupg/secring.gpg
+
+ossrhUsername=use maven repo username
+ossrhPassword=use maven repo password
```

Comment:
I think we should exclude the password from the repo, we can inject it during the build/publish actions